nestjs/README.md

```diff
@@ -0,0 +1,4 @@
+# NestJS example
+
+From the nestjs folder you can run with `bazel run //:bin` and then perform a `GET` on http://localhost:3000/hello-world
```

Comment:
Probably drop the "from the nestjs folder". I think each example assumes you'll be within the the WORKSPACE directory